### ISSUE TITLE
Cannot install unpackaged app after a packaged app has been installed

### ISSUE THREAD

**Issue by andrewleader** [microsoft/WindowsAppSDK#1871]
### Describe the bug

The WinAppRuntime Installer fails to install if a packaged app using WinAppSDK has already been installed. It seems as if fails while trying to update the existing packages and therefore fails to install the remaining packages that are needed for an unpacakged app.

### Steps to reproduce the bug

First, **uninstall all versions of WinAppRuntime** by using an admin PowerShell to replicate a "clean" computer...

```
$winappsdk = "Microsoft.WindowsAppRuntime.1.0*"
Get-AppxPackage | Where-Object { $_.Dependencies -like $winappsdk } | Remove-AppxPackage
Get-AppxPackage $winappsdk | Remove-AppxPackage
```

Then, **install a MSIX WinUI 3 app** like https://github.com/andrewleader/WindowsAppSDKGallery/releases/download/v0.39.1.0/WindowsAppSDKGallery_0.39.1.0_AnyCPU.msix (you'll have to add the certificate before installing)

Then, **try running the x64 Installer** to prep for using an unpackaged app: https://aka.ms/windowsappsdk/1.0-stable/msix-installer

Notice the following errors appear...

```
Installing license: WAR_MAINPACKAGE_LICENSE
Install result : 0x0
Deploying package: Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x64__8wekyb3d8bbwe
Package deployment result : 0x80073cfb The provided package is already installed, and reinstallation of the package was blocked. Check the AppXDeployment-Server event log for details.
One or more install operations failed. Result: 0x80073cfb

[process exited with code 2147958011]
```

**Try running an unpackaged app** like https://github.com/andrewleader/WindowsAppSDKGallery/releases/download/v0.39.1.0/WindowsAppSDKGallery-unpackaged-0.39.1.0.zip (extract the ZIP and then double click `WindowsAppSDKGallery.exe`)

**Notice the EXE crashes** (nothing visible appears since fails before XAML starts up).

**Uninstall all WinAppRuntime** using the same PowerShell script at the start, and then **try running the x64 Installer again** and try using the unpackaged app again. Notice this time it works fine.

### Expected behavior

Should be able to install unpackaged apps regardless of whether packaged apps have already been installed.

### NuGet package version

1.0.0

### Packaging type

Unpackaged

### Windows version

Insider Build (xxxxx)

### IDE

Visual Studio 2022

**Comment by DrusTheAxe** [comment]
>Then, install a MSIX WinUI 3 app...
...
Then, try running the x64 Installer to prep for using an unpackaged app: https://aka.ms/windowsappsdk/1.0-stable/msix-installer

Please run the following command between these steps:

```
powershell -c $(Get-AppxPackage microsoft*win*app*run*).PackageFullName
```

Given the installer reported

```
Deploying package: Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x64__8wekyb3d8bbwe
...
...The provided package is already installed...
```

I'd expect Microsoft.WindowsAppRuntime.1.0 with a version >= 0.319.455.0 is already registered when you run the installer.

If so this is a known issue in the installer (should detect this condition and treat as success).

@sachintaMSFT I thought we had a work item for that but can't find it to share here via https://task.ms/nnnnnn

**Comment by headsea** [comment]
Any update? I can't uninstall and install...

Microsoft Visual Studio Professional 2019 Version 16.11.8

`Severity	Code	Description	Project	File	Line	Suppression State
Error		DEP0800: The required framework "C:\Users\xxx\.nuget\packages\microsoft.windowsappsdk\1.0.0\build\..\tools\MSIX\win10-x64\Microsoft.WindowsAppRuntime.1.0.msix" failed to install. 
[0x80073CFB] Deployment of package Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x64__8wekyb3d8bbwe was blocked because the provided package has the same identity as an already-installed package but the contents are different. Increment the version number of the package to be installed, or remove the old package for every user on the system before installing this package.	Shell (Package)			
`

**Comment by davidrios** [comment]
I have the same issue.

The results of the suggested command the first time
```
> powershell -c $(Get-AppxPackage microsoft*win*app*run*).PackageFullName
Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x64__8wekyb3d8bbwe
Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x86__8wekyb3d8bbwe
```

After uninstalling all WinAppRuntime and running the x64 Installer again:
```
> powershell -c $(Get-AppxPackage microsoft*win*app*run*).PackageFullName
Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x64__8wekyb3d8bbwe
Microsoft.WindowsAppRuntime.1.0_0.319.455.0_x86__8wekyb3d8bbwe
MicrosoftCorporationII.WindowsAppRuntime.Main.1.0_0.319.455.0_x64__8wekyb3d8bbwe
Microsoft.WindowsAppRuntime.Singleton_0.319.455.0_x64__8wekyb3d8bbwe
Microsoft.WinAppRuntime.DDLM.0.319.455.0-x6_0.319.455.0_x64__8wekyb3d8bbwe
Microsoft.WinAppRuntime.DDLM.0.319.455.0-x8_0.319.455.0_x86__8wekyb3d8bbwe
```

**Comment by DrusTheAxe** [comment]
We've identified the root problem and working on a fix. Go [here](https://github.com/microsoft/WindowsAppSDK/issues/1953#issuecomment-1039889981) for more details.

**Comment by riverar** [comment]
@DrusTheAxe While that's the root cause, this issue demonstrates that the installer doesn't swallow package-already-installed HRESULTs. Perhaps consider doing so to cover the scenario in which the user is legitimately blocked from reinstalling that package?

**Comment by DrusTheAxe** [comment]
> demonstrates that the installer doesn't swallow package-already-installed HRESULTs.

It's not quite that simple. The installer _does_ detect already-installed and if so, re-register the package (a form of repair-if-something's-broken). That's detected just fine as not an error, as you can see [here](https://github.com/microsoft/WindowsAppSDK/blob/7338d01099178207f000d9c3bab8e8806afeb9f0/installer/dev/install.cpp#L48)

The problem here's slightly different - package XYZ is installed and you're trying to install a package with the same identity _but different content_. That violates a base principle of the appmodel -- package identity is unique across space and time. That multiple packages with the same package identity _has the same bits_.

The bug here is the 1.0.0 framework packages available for download and included in the NuGet, VSIX and installer are identical _but the frameworks installed by the Store have different content_. Specifically, their `appxmanifest.xml` differs - BOM vs no BOM, added `<mp:PhoneIdentity>`, formatted differently. The manifest's file hash differs because the files' aren't identical, thus same identity with different content. Deployment detects that inconsistency and fails the install.

The root problem is due to a bug in our build process. We've corrected that so it won't happen again. This was specific to 1.0.0. Anyone with (any) 1.0.0 packages installed can upgrade fine to 1.0.1(+) packages just fine. It's only the "install the same thing that's not really the same thing" that's a problem.

**Comment by gabbybilka** [comment]
Fixed in 1.0.1, check out the release for more info: #2291
